server/oresat-live-software-rx.service

```diff
@@ -7,7 +7,7 @@ User=root
 Group=root
 WorkingDirectory=/usr/local/sbin/oresat-live-software-server/
 ExecStartPre=mkdir -p public/video
-ExecStart=/usr/bin/bash rx.sh
+ExecStart=/usr/bin/bash rx.sh mon0 ./public/video/
```

Comment:
Normally I'd be a little bit concerned about hard-coding `mon0` come to think of it. Nothing actually guarantees that's a valid device.

This is probably fine for our purposes since we're using an image, where the relevant network configuration file presumably exists. Ideally something in `rx.sh` would validate the device and maybe pick a valid default one.